src/libraries/System.IO.Compression/src/System/IO/Compression/ZipBlocks.cs

```diff
@@ -503,7 +506,7 @@ public static void WriteBlock(Stream stream, long numberOfEntries, long startOfC
         }
     }
 
-    internal readonly partial struct ZipLocalFileHeader
+    internal sealed partial class ZipLocalFileHeader
```

Comment:
This class effectively has no state at all (just a lot of `static` methods and `public consts` and two `public static` members (for the signature bytes.  It was previously a `readonly struct` so there's no real possibility of mutable state... so why do we need to convert this to a class?

Reply:
Good catch, I should revert this for ZipLocalFileHeader.